src/assets/scss/components/_products.scss

```diff
@@ -387,6 +390,27 @@
   @include breakpoint-down(mobile) {
     padding: 1rem 1.25rem 2rem;
   }
+  .success-tldr-wrapper {
+    display: flex;
+    flex-direction: row;
+    max-width: 950px;
+    margin: 2rem auto 0;
+    background-color: #f5f5f5;
+    div {
+      flex-basis: 20%;
+    }
+    img {
+      // height: 100px;
+      margin-left: 3rem;
+      margin-top: -1rem;
+    }
+    p {
+      flex-basis: 75%;
+      font-family: "Halyard Display";
```

Comment:
i think this should be inherited and can be removed

Reply:
Fair! done.